react-form-controls/my-app/src/RegistrationFormUncontrolled.jsx

```diff
@@ -0,0 +1,28 @@
+export default function RegistrationFormUncontrolled() {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const usrnm = document.querySelector('#usrnm').value;
```

Comment:
I would use `event.target` instead to avoid unnecessary DOM transversals as pass that instead to create your `formData`